superset-frontend/src/SqlLab/components/SqlEditor/index.jsx

```diff
@@ -354,39 +355,28 @@ const SqlEditor = ({
     return base;
   }, [dispatch, queryEditor.sql, startQuery, stopQuery]);
 
-  const handleWindowResize = useCallback(() => {
+  const handleWindowResize = useEffectEvent(() => {
     setHeight(getSqlEditorHeight());
```

Comment:
```suggestion
    setHeight(height);
```

Reply:
In this approach, this doesn't need to wrap with useEffectEvent since no references needed other than setHeight.
Let me clean up with setHeight then.